src/test/java/org/minions/devfund/noemiguzman/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,233 @@
+package org.minions.devfund.noemiguzman.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+
+import static junit.framework.TestCase.assertTrue;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+/**
+ *  Class to test movie database.
+ */
+public class MovieDatabaseTest {
+    private MovieDatabase mdb1;
+    private MovieDatabase mdb2;
+    private MovieDatabase mdb3;
+
+    private Movie movie1;
+    private Movie movie2;
+    private Movie movie3;
+
+    private ArrayList<Movie> movieList1;
+    private ArrayList<Movie> movieList2;
+    private ArrayList<Movie> movieList3;
+
+    private Actor actor1;
+    private Actor actor2;
+    private Actor actor3;
+
+    private ArrayList<Movie> movieList;
+    private ArrayList<Actor> actorList;
+
+    /**
+     * Method to setup movie1 and movie2.
+     */
+
+    @Before
+    public void setUpMovieDatabases() {
+        mdb1 = new MovieDatabase();
+        mdb2 = new MovieDatabase();
+        mdb3 = new MovieDatabase();
+
+        movie1 = new Movie("movie1");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "movie1" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)